src/api/app/views/webui2/webui/patchinfo/edit.html.haml

```diff
@@ -0,0 +1,47 @@
+- @pagetitle = "Edit Patchinfo for #{@project}"
```

Comment:
Ok as User, there are some issues with that page that may or may not be addressed.

Nobody edit the issues summary since it comes from Bugzilla, and there are some internal tools which match the issues summary from patchinfo with bugzilla issues. AFAIK, L3 does it. What users can do is remove an issue or add new issues.

The binaries section, could be hidden under "advanced options" if possible. I don't remember a case where we use it consistently.

Better align the fields to save some spaces and have description as text area with enough space to add long texts. Normally patchinfos are really verbose, with many issues (specially true for SLE updates) . The "old view" is actually "well designed" (thinking about maintenance team usability)

It would be great to have the name of the package for which the patchinfo is being written. For example this incident https://build.suse.de/project/show/SUSE:Maintenance:10415 has 3 packages.

Reply:
Regarding editing the description of an issue, I kept the same functionality, just made it obvious to do it. It was previously possible to edit the description, but you had to know that clicking on the description would display an input which then allow you to edit the description. So for now, I would keep this functionality until we redesign this page/workflow (if we ever do it).

For the binaries section, I would not hide a section of a form. If this section is not needed in some patchinfo, the field is not required, so it can be ignored. This goes back to the possible redesign of this page/workflow.

Concerning this paragraph:

> Better align the fields to save some spaces and have description as text area with enough space to add long texts. Normally patchinfos are really verbose, with many issues (specially true for SLE updates) . The "old view" is actually "well designed" (thinking about maintenance team usability)

I'm all ears, but please, be clear on what is to be improved. What do you mean with better align the fields to save some spaces? The fields are fully using the available space (in comparison to the old view, which didn't at all). The `textarea` elements now have a few rows (thanks to @DavidKang for his commit). As for the old view is well designed, I basically took the form from the old view and kept the same functionality, then added a bunch of UX improvements (they're listed in the PR's description). Heck, the old view didn't even have any indication to show what is required (which is basic design IMHO).

About having the name of the package(s) for which the patchinfo is being written, the patchinfo is for a project, not for a package. The project will contain the patchinfo and packages. I would consider this feature for a redesign, not for this migration.